swagger2.0/oic.r.cloudopenapi.swagger.json

```diff
@@ -722,49 +722,50 @@
       }
     },
     "DeviceWithLinks": {
```

Comment:
It's DeviceInformation , links / no links version no more exists.

Reply:
Changed to align with the single definition of /device in the primary branch